don't force C11, especially since nothing seems to require it, at the moment

some typedef redefinitions are only allowed in C11 though (at least in theory),
but it's easy to undo them.

this avoids having any requirement on an updated libc on older OSX

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -25,7 +25,7 @@
 set(CMAKE_CXX_STANDARD 14)
 set(CMAKE_CXX_STANDARD_REQUIRED TRUE)
 
-set(CMAKE_C_STANDARD 11)
+set(CMAKE_C_STANDARD 99)
 set(CMAKE_C_STANDARD_REQUIRED TRUE)
 set(CMAKE_C_VISIBILITY_PRESET hidden)
 set(CMAKE_VISIBILITY_INLINES_HIDDEN TRUE)
--- a/arm-wt-22k/lib_src/eas_filter.h
+++ b/arm-wt-22k/lib_src/eas_filter.h
@@ -2,8 +2,6 @@
 #include "eas_options.h"
 #include "eas_types.h"
 
-typedef struct s_wt_int_frame_tag S_WT_INT_FRAME;
-
 #if defined(_FILTER_ENABLED)
 /*----------------------------------------------------------------------------
  * S_FILTER_CONTROL data structure
--- a/arm-wt-22k/lib_src/eas_miditypes.h
+++ b/arm-wt-22k/lib_src/eas_miditypes.h
@@ -31,10 +31,9 @@
 #ifndef _EAS_MIDITYPES_H
 #define _EAS_MIDITYPES_H
 
+#include "eas_synth.h"
 #include "eas_parser.h"
 
-typedef struct s_synth_tag S_SYNTH;
-
 /*----------------------------------------------------------------------------
  * S_MIDI_STREAM
  *
--- a/arm-wt-22k/lib_src/eas_wtengine.h
+++ b/arm-wt-22k/lib_src/eas_wtengine.h
@@ -41,7 +41,6 @@
 
 #include "eas_sndlib.h"
 #include "eas_wt_IPC_frame.h"
-#include "eas_filter.h"
 
 /*----------------------------------------------------------------------------
  * defines
@@ -70,6 +69,7 @@
     EAS_I32         prevGain;
 } S_WT_INT_FRAME;
 
+#include "eas_filter.h"
 
 /*------------------------------------
  * S_LFO_CONTROL data structure
